conf/distro/nilrt.inc

```diff
@@ -141,6 +142,10 @@ NILRT_FEEDS_URI_RELEASE            ?= "${NILRT_FEEDS_URI}/${NILRT_FEED_NAME}"
 NILRT_MACHINE_FEED_URI_x64         ?= "${NILRT_FEEDS_URI_RELEASE}/x64"
 NILRT_MACHINE_FEED_URI_xilinx-zynq ?= "${NILRT_FEEDS_URI_RELEASE}/arm"
 
+NILRT_MAIN_FEED_VERSION            ?= "2020.7"
```

Comment:
[Follow-up Item]
I assume you're already tracking this: but let's remember to bump this version to the most-current feed on nickdanger.